.github/workflows/sample.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     - name: 'Deploy to Azure Container Apps'
       uses: ./
       with:
-        resource-group: rg-hoge
-        name: ctapp-hoge
+        resource-group: rg-sample
+        name: ctapp-sample
         image: mcr.microsoft.com/azuredocs/containerapps-helloworld:latest
+        revision-name-suffix: ${{ github.sha }}
```

Comment:
In case of deactivate, is it correct to write sample.yml as follows?

```YAML
    - name: Deactivate Preview Deployment
        if: github.event.action == 'closed'
        uses: ./
        with:
                mode: deactivate-revision
                resource-group: rg-sample
                name: ctapp-sample
                image: mcr.microsoft.com/azuredocs/containerapps-helloworld:latest
                revision-name-suffix: ${{ github.sha }}
```

Reply:
Yes, and I want to discuss input parameters in this mode.